### ISSUE TITLE
play local video in app

### ISSUE THREAD

**Issue by ibrha** [lipangit/JiaoZiVideoPlayer#2524]
hi i use last update library when i play video in assets its not working just rotate the loader icon 

my code for playing

 try{
            JZDataSource jzDataSource ;
            jzDataSource = new JZDataSource(Uri.parse("file:///android_asset/video.mp4");
            jzDataSource.title = "your title";
            jcVideoPlayerStandard.setUp(jzDataSource, JzvdStd.SCREEN_WINDOW_NORMAL);
            jcVideoPlayerStandard.startVideo();
        }catch (Exception e){
            Log.d("logloglog", e.toString());
        } 
its first time to play a video from local apk

**Comment by lipangit** [comment]
![image](https://user-images.githubusercontent.com/2038071/61207155-eee39980-a726-11e9-96be-6b6a37859f79.png)

see it in demo

**Comment by ibrha** [comment]
is that mean i need another ibrary to play a local from app video like import tv.danmaku.ijk.media.player.IMediaPlayer;
import tv.danmaku.ijk.media.player.IjkMediaPlayer;
import tv.danmaku.ijk.media.player.IjkTimedText; ?

**Comment by lipangit** [comment]
no

**Comment by ibrha** [comment]
sorry but i didnt get it ,, what should i do with that class ?

**Comment by lipangit** [comment]
copy from demo

**Comment by ibrha** [comment]
thank you its work but its just the sound of video playing ! and circle rotating 

the code : 
    JzvdStd jzvdStd =  findViewById(R.id.jz_video);

  JZDataSource jzDataSource = null;
    try {
        jzDataSource = new JZDataSource(getAssets().openFd("bts.mp4"));
        jzDataSource.title = "饺子快长大";
        jzvdStd.setUp(jzDataSource, JzvdStd.SCREEN_NORMAL, JZMediaSystemAssertFolder.class);
        jzvdStd.startVideo();
    } catch (IOException e) {
        e.printStackTrace();
    }
![image](https://user-images.githubusercontent.com/31315586/61622282-26010000-ac7d-11e9-95b8-41b5ab9fba42.png)

**Comment by lipangit** [comment]
see JZMediaSystemAssertFolder in  develop branch of the new version

**Comment by ibrha** [comment]
yes i get that class from branch develop  Updated 8 days ago by lipangit

**Comment by ibrha** [comment]
yes i get that class from branch develop Updated 8 days ago by lipangit but its same like a image up

**Comment by ibrha** [comment]
no answers ?

**Comment by lipangit** [comment]
![image](https://user-images.githubusercontent.com/2038071/61866963-c0c13f00-af08-11e9-8225-8f6072eae349.png)


demo works good. You should make sure your source code is same as the demo source code.
